src/vaults/factories/PoolEscrowFactory.sol

```diff
@@ -34,6 +37,7 @@ contract PoolEscrowFactory is IPoolEscrowFactory, Auth {
         PoolEscrow escrow_ = new PoolEscrow{salt: bytes32(uint256(poolId))}(poolId, address(this));
 
         escrow_.rely(root);
+        escrow_.rely(gateway);
         escrow_.rely(poolManager);
         escrow_.rely(balanceSheet);
         escrow_.rely(asyncRequests);
```

Comment:
Note to myself: Missing test coverage for deployed escrow wards

Reply:
They are already added to the wards IIUC your note